packages/lodestar-beacon-state-transition/src/fast/epoch/processRewardsAndPenalties.ts

```diff
@@ -24,6 +24,7 @@ export function processRewardsAndPenalties(epochCtx: EpochContext, process: IEpo
       newBalances[i] -= penalty;
     }
   });
+  process.balances = newBalances;
```

Comment:
For a second I thought we are adding stuff the global NodeJS `process` object xD I another PR may we want to change the name so it doesn't match any existent global variable?